server/queue/queue-jms/src/test/java/org/apache/james/queue/jms/LegacyJMSMailQueueTest.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.james.queue.jms;
+
+import java.util.Arrays;
+
+import org.apache.activemq.broker.BrokerService;
+import org.apache.activemq.broker.region.policy.PolicyEntry;
+import org.apache.activemq.broker.region.policy.PolicyMap;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+public class LegacyJMSMailQueueTest extends AbstractJMSMailQueueTest {
```

Comment:
Is there any reason to keep it at the end of this PR?

Reply:
Yes. It is needed for the ActiveMQ test class. It is resolved here: https://github.com/linagora/james-project/pull/1228